server/mobile-services.js

```diff
@@ -134,6 +159,15 @@ const PushService = {
               };
             }
 
+            // get the AndroidVariant and add it
+            const webPushVariant = variants.find(v => !!v && v.kind === WEB_PUSH_VARIANT_KIND);
```

Comment:
The `!!v &&` is there in the `!!v && v.kind` just to ensure that `v` is not null. Since we are not returning the value, there is no advantage in making the double negation. This can simply be `v && v.kind

```suggestion
            const webPushVariant = variants.find(v => v && v.kind === WEB_PUSH_VARIANT_KIND);
```

Reply:
The check `!!v` is also done on both Android and iOS variants. Should I remove this from them.